srv/modules/runners/openstack.py

```diff
@@ -0,0 +1,212 @@
+# -*- coding: utf-8 -*-
+# vim: ts=8 et sw=4 sts=4
+"""
+Wrapper to apply the ceph.openstack state to the salt master, then
+return the ceph configuration and keys necessary to subsequently
+configure OpenStack.
+"""
+
+from __future__ import absolute_import
+import re
+import salt.client
+
+try:
+    import configparser
+except ImportError:
+    import salt.ext.six.moves.configparser as configparser
+
+
+def integrate(**kwargs):
+    """
+    Create pools and users necessary for OpenStack integration.  Returns
+    relevant Ceph configuration and keys, to be used to subsequently
+    configure OpenStack.
+
+    This will create pools for use by glance, cinder, cinder-backup and nova.
+    By default, these pools will be named "images", "volumes", "backups" and
```

Comment:
I think the prefix=cloud should be default here. That makes it quite clear in the first place. We still can make the prefix option available for let's say dc2-cloud-images (datacenter2).